### DISCUSSION TITLE
Problem with dropdown select

### DISCUSSION THREAD

**Discussion by bobimicroweber** [livewire/livewire#4894] [1. Help]
When i have only one option in select field, the value of the select is not sended to backend.
`
<select class="form-control" wire:model="import_feed.content_tag" id="feed_content_tag">
                                        @foreach($this->import_feed['detected_content_tags'] as $contentTagKey=>$contentTagVal)
                                            <option @if($this->import_feed['content_tag'] == $contentTagKey) selected="selected" @endif value="{{$contentTagKey}}">{{$contentTagKey}}</option>
                                        @endforeach
                                    </select>
`

This foreach has one item and when i click save this field has no value in backend.

**Comment by tushargugnani** [comment]
@bobimicroweber The default value / first value shown in the select box will not automatically binded to the property. You have two options.
1. Use the mount method to set the default value to the backend property import_feed.content_tag
2. Give a select option to the user to force selecting a value before form submission.
